apps/backend/src/experiments/core/repositories/experiment-member.repository.ts

```diff
@@ -19,12 +19,41 @@ export class ExperimentMemberRepository {
   async getMembers(
     experimentId: string,
   ): Promise<Result<ExperimentMemberDto[]>> {
-    return tryCatch(() =>
-      this.database
-        .select()
+    return tryCatch(async () => {
+      const rows = await this.database
+        .select({
+          experimentId: experimentMembers.experimentId,
+          userId: experimentMembers.userId,
+          role: experimentMembers.role,
+          joinedAt: experimentMembers.joinedAt,
+          user: {
+            name: sql<string>`COALESCE(${users.name}, '')`.as("name"),
+            email: users.email,
+          },
+        })
         .from(experimentMembers)
-        .where(eq(experimentMembers.experimentId, experimentId)),
-    );
+        .innerJoin(users, eq(experimentMembers.userId, users.id))
+        .where(eq(experimentMembers.experimentId, experimentId));
+
+      // Map: ensure correct types and fields
+      return rows.map((row) => {
+        if (!row.userId) {
+          throw new Error(
+            "User not found for experiment member: " + JSON.stringify(row),
+          );
+        }
+        return {
+          experimentId: row.experimentId,
+          userId: row.userId,
+          role: row.role,
+          joinedAt: row.joinedAt,
+          user: {
+            name: row.user.name,
+            email: row.user.email ?? null,
+          },
+        };
+      });
+    });
```

Comment:
this mapping should not be needed     if (!row.userId) {
          throw new Error(
            "User not found for experiment member: " + JSON.stringify(row),
          );
        }this cant happen since userId is a required in the table